package/android/cpp/rnskia-android/SkiaOpenGLHelper.h

```diff
@@ -0,0 +1,246 @@
+#pragma once
+
+#include "EGL/egl.h"
+#include "GLES2/gl2.h"
+#include <fbjni/fbjni.h>
+#include <jni.h>
+
+#include "RNSkLog.h"
+
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdocumentation"
+
+#include "SkCanvas.h"
+#include "SkColorSpace.h"
+#include "SkSurface.h"
+#include "include/gpu/GrDirectContext.h"
+#include "include/gpu/gl/GrGLInterface.h"
+
+#pragma clang diagnostic pop
+
+namespace RNSkia {
+struct SkiaOpenGLContext {
+  SkiaOpenGLContext() {
+    glDisplay = EGL_NO_DISPLAY;
```

Comment:
these default values are never used I think.
Also the display is only needed in the "top-level" context.

Reply:
We use the default values to test if the context is initialised. In release mode you cannot be guaranteed that structs etc are initialised to null / zero.

Display is only top-level - but we keep it as a thread value so that we don't need any locking to access the value from different threads. It is faster and cheaper.